templates/telco/index.html

```diff
@@ -0,0 +1,242 @@
+                    </p>
+                    <p>
+                        <a href="">
+                            Learn about value-added telco services with Anbox Cloud >
+                        </a>
+                    </p>
+                </div>
+                <div class="col-6 u-align--center u-hide--small">
+                    {{ image (
+                        url="https://assets.ubuntu.com/v1/d5de891a-Acceleration.svg",
+                        alt="",
+                        width="287",
+                        height="160",
+                        hi_def=True,
+                        loading="auto"
+                        ) | safe
+                    }}
+                </div>
+            </div>
+        </section>
+
+        <section class="p-strip">
+            <div class="row u-vertically-center">
+                <div class="row col-7">
+                    <div class="col-9">
+                        <h2>Inventing value-added services for 5G</h2>
+                    </div>
+                    <p>
+                        5G makes ultra-rich media content, games, AR/VR, and even AI, accessible within telco RANs. With unrestricted computing capabilities available on demand, telcos can create innovative subscriber experiences across various mobile form factors to offer differentiated services to their subscribers.
+                    </p>
+                    <p>
+                        What does it take to realise this vision? Considering the scale, this first and foremost requires servers in the cloud with an outstanding cost to performance ratio, equipped with application accelerators like GPUs, FGPAs and ASICs. Furthermore, to minimise latency, mobile workloads should be deployed at the edge, closer to mobile subscribers. Finally, instant elasticity and scalability are required to accommodate subscriber mobility patterns.
+                    </p>
+                </div>
+                <div class="col-5">
+                    <div class="p-card">
+                        <h4>
+                            Key challenges:
+                        </h4>
+                        <hr class="u-sv1">
+                        <ul class="p-list">
+                            <li class="p-list__item is-ticked">Access to elastic compute </li>
+                            <li class="p-list__item is-ticked">Deployment agility into edge RANs</li>
+                            <li class="p-list__item is-ticked">Multi-cloud software automation</li>
+                        </ul>
+                        <p>
+                            Anbox Cloud is engineered to address these challenges. The commercially supported platform builds on open source software to empower innovators to leverage Android at scale in the cloud.
+                        </p>
+                    </div>
+                </div>
+            </div>
+            <div class="p-strip">
```

Comment:
We don't normally put a `p-strip` inside another `p-strip` - the `row` class should be enough here